app/models/seed_trade.rb

```diff
@@ -0,0 +1,46 @@
+class SeedTrade < ActiveRecord::Base
+  belongs_to :seed
+  belongs_to :requester, class_name: 'Member'
+
+  default_scope { order(created_at: :desc) }
+
+  # List seed trades both when member is requester and seed owner
+  scope :by_member,
+                 ->(member) { includes(:seed).where(
+                 'seeds.owner_id = ? OR requester_id = ?',
+                 member.id, member.id).references(:seeds) }
```

Comment:
I think I understand your concern, when we'll have many records would be more user friendly to show by default only "not replied" requests, and because of this we should think about the better way (in performance terms) to nest not_replied and by_member scopes results, right?